### PR TITLE
add published json file to populate cvee feed on the website

### PR DESCRIPTION
**Pull request by cartersocha**
Re-uses assets from the private repo to create a public json file of published security incidents

### CODE REVIEW

**Comment by cartersocha** [comment]
the token secret will need to be added as a repo secret for this to work

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+      - name: setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9' # install the python version needed
+          
+      - name: install python packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
```
Comment (codeboten) [inline .github/workflows/actions.yml:25]: Does this file need to be checked in as well?

**Review by codeboten — COMMENTED** [review]
Just one question... I would suggest we should enable code scanning in the repo as well once we merge code in :)

**Comment by cartersocha** [comment]
> Just one question... I would suggest we should enable code scanning in the repo as well once we merge code in :)

my bitcoin mining is necessary for security

**Review by codeboten — APPROVED** [review]

**Pull request merged**